hydra_pspec/pspec.py

```diff
@@ -1,12 +1,57 @@
 import numpy as np
 import scipy as sp
+from scipy.interpolate import interp1d
 from scipy.stats import invgamma
 
 from multiprocess import Pool, current_process
 from . import utils
 import time
 
 
+def inversion_sample_invgamma(alpha, beta, prior_min, prior_max, ngrid=1000):
+    """
+    Draw a sample from an inverse gamma distribution between prior bounds using
+    inversion sampling.
+    
+    This works by sampling the cdf of the inverse gamma distribution 
+    on a (logarithmic) grid and then interpolating to convert a uniform 
+    random draw into a random draw with the correct PDF.
+
+    Parameters:
+        alpha (float):
+            Inverse gamma alpha parameter.
+        beta (float):
+            Inverse gamma beta (scale) parameter.
+        prior_min (float):
```

Comment:
Clarify that "The log10() of this value will be taken"

Reply:
Updated docstrings for both prior bounds